drivers/display/ssd1327.c

```diff
@@ -56,8 +69,8 @@ static inline int ssd1327_write_bus_cmd(const struct device *dev, const uint8_t
 		return err;
 	}
 	for (size_t i = 0; i < len; i++) {
-		err = mipi_dbi_command_write(config->mipi_dev, &config->dbi_config,
-					     data[i], NULL, 0);
+		err = mipi_dbi_command_write(config->mipi_dev, &config->dbi_config, data[i], NULL,
```

Comment:
Try not to add any formatting changes.

Reply:
yeah I know. This is an age old discussion. Generally speaking for completely new code people tend to use clang-format. For working on existing code that has been written without it, generally it is preferred to leave the code as is, because it makes patches less readable/reviewable. I know, that it's messy because on code that you added to an existing file you do want  some formatting. But yeah.